File: c-006/Main.java
import java.util.stream.IntStream;

public class Main {

    public static int sumDifSquare(int num) {
        int x = IntStream.rangeClosed(1, num)
            .map(a -> (int) Math.pow(a, 2))
            .sum();
        int y = IntStream.rangeClosed(1, num)
            .sum();
        return (int) Math.pow(y, 2) - x;
    }
    
    public static void main(String[] args) {
        System.out.println(sumDifSquare(100));
    }
}

File: c-006/main.rs
fn sum_dif_square(num: i32) -> i32 {
    let x: i32 = (1..(num + 1))
        .map(|a| i32::pow(a, 2))
        .sum();
    let y: i32 = (1..(num + 1))
        .sum();
    return i32::pow(y, 2) - x;
}

fn main() {
    println!("{}", sum_dif_square(100));
}
